src/store/actions/profile.action.js

```diff
@@ -0,0 +1,13 @@
+import { createAsyncThunk } from "@reduxjs/toolkit";
+import axios from 'axios'
+
+
+const fetchProfileData = createAsyncThunk(
+    'profile/fetchProfileData',
+    async () => {
+        const res = await axios.get('https://api.github.com/user')
```

Comment:
Why don't we abstract the baseUrl `https://api.github.com` and make it an environment variable in `.env`